### PR TITLE
Create bigtime.svg

### PR DESCRIPTION
**Pull request by abalone1969**
new to this world, 
peace
HipstamaticPhoto-594340235 286683

### CODE REVIEW

**Comment by neauoire** [comment]
I love it.

**Comment by abalone1969** [comment]
fantástico 
thank you

**Comment by neauoire** [comment]
Could you fix it tho? Just close the </svg> tag

**Comment by abalone1969** [comment]
right on
thank you

I pushed the ‘close pull request button

stoked to be learning so much

grateful for your help

**Comment by neauoire** [comment]
I didn't mean for you to close the Pull Request, I meant, in your theme file, you forgot to close the `<svg>`, you need to close it with `</svg>` on the last line.

**Comment by abalone1969** [comment]
comedy 
and growth

File: bigtime.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  <!-- Foreground -->
+  <circle cx='24' cy='24' r='8' id='f_high' fill='#FFA500'></circle>
+  <circle cx='40' cy='24' r='8' id='f_med' fill='#2F4F4F'></circle>
+  <circle cx='56' cy='24' r='8' id='f_low' fill='#000000'></circle>
+  <circle cx='72' cy='24' r='8' id='f_inv' fill='#9932CC'></circle>
+  <!-- Background -->
+  <circle cx='24' cy='40' r='8' id='b_high' fill='#696969'></circle>
+  <circle cx='40' cy='40' r='8' id='b_med' fill='#778899'></circle>
+  <circle cx='56' cy='40' r='8' id='b_low' fill='#F8F8FF'></circle>
+  <circle cx='72' cy='40' r='8' id='b_inv' fill='#6B8E23'></circle>
```
[the remark this answers is not in the document]
Comment (abalone1969) [reply bigtime.svg]: gracias, I have a lot to learn...

**Pull request merged**